test/powershell/engine/Remoting/InvokeCommandRemoteDebug.Tests.ps1

```diff
@@ -0,0 +1,200 @@
+﻿##
+## PowerShell Invoke-Command -RemoteDebug Tests
+##
+
```

Comment:
This comment doesn't block the PR merge.
But I do think for features like this one, we should at least verify if it works on NanoServer, because it adds great value to customers and thus is very possible to be cherry-picked to the NanoServer in-box powershell.